### DISCUSSION TITLE
乞丐的收入算主動收入還是被動收入

### DISCUSSION THREAD

**Discussion by roma0323** [1101-GS4719-C/Python-Community-GS4719-C#1514] [Q&A]
上網查了一些定義：不用上班賺錢，每天不用做事就會自動產生的收入。
不工作也能賺到的現金流收入。

那乞丐坐在地上等別人來給錢算是在工作嗎？
我自己覺得是兩個都算啦，但是想聽聽大家的想法

**Comment by Chiuhsianglin** [comment]
我發現我自己好像歸不出來算哪一種收入，於是我問了chat gpt，他說乞丐的收入屬於主動收入。主動收入是指透過個人勞力、工作或提供服務所獲得的收入。在這種情況下，乞丐透過向他人乞討或提供某種形式的服務（例如清潔窗戶、搬運行李等）來賺取收入。乞丐需要主動參與和努力工作才能獲得收入，因此可以歸類為主動收入。

**Comment by poweichen00** [comment]
我覺得對於某些人來說，乞討並不符合傳統的工作定義，因為它沒有明確的工作職責和報酬，而對於其他人來說，乞討可以被視為一種努力尋求收入的工作。

**Comment by PaulSu0905** [comment]
我認為應該是主動收入，因為現在外面很多的乞丐在乞討時都會主動出擊，例如直接走到你面前要求金錢幫助，或是做一些乞討的動作，而不只是坐在原地等人給錢。所以我認為他是有付出勞力來換取金錢的行為。

**Comment by youtinglin** [comment]
我認為是主動收入，因為他不是別人主動給他的，而是他在街上主動和別人乞討，所以也算有付出行動吧。

**Comment by Boying12** [comment]
雖然不是很歡這個工作(? ，但根據定義，儘管乞丐的收入可能相對不穩定，且受多種因素影響（如天氣、位置、人流量等），但由於他們需要不斷地去乞討以獲得收入，這種收入仍然屬於主動收入的範疇。

**Comment by poyen0806** [comment]
我認為乞丐在某些層面來看，他或許算是個表演藝術家，如前面同學所說，該職業受多種因素影響（如天氣、位置、人流量等），即使在一個好的環境，也要表現出令人同情的模樣才能得到收入，且演技與收入呈正相關，所以我認為該職業的收入屬主動收入。

**Comment by 112121py** [comment]
在某種程度上也許算是一種主動收入，他們有點像放下身段（？，請求別人的幫助，不過我的想法是建立在他們有主動發出要求這件事，如果只是單純的坐著躺著，也許不那麼算是「主動」嗎，是與否好像還要看要怎麼界定啦

**Comment by qiu0109** [comment]
我覺得應該算是主動收入，因為乞丐除了坐在地上，還會有向人請求的動作，也算是有付出行動。

**Comment by 113607514** [comment]
我覺得乞丐的收入應該算是主動收入吧。因為他們的收入基本上是靠每天的乞討和與別人互動來賺的，得靠自己去做事，像是去街上乞討、跟人說話、展示需求，這些都需要主動去做，收入也會根據他們當天的行為而變動。所以這不像是被動收入，因為被動收入是那種不用自己每天做事情也會進來的錢，像是租金或投資回報。對我來說，乞丐的收入是完全依賴他們的日常努力和時間，應該算主動收入。

**Comment by DarsooK** [comment]
乞丐的收入更接近主動收入，因為他們需要出現特定地點、花時間乞討，這是一種依賴主觀行為和付出（即使是存在本身）的賺錢方式。如果完全什麼都不做還能有錢進帳，那才算是被動收入。

**Comment by yue083139** [comment]
我覺得乞丐還是算主動收入，像我之前在一中街，就能看到各種乞丐們各顯神通，有些可能是躺在大馬路中間做行為藝術，有些則會跑到公車站牌硬是要找人搭話給予情緒價值（負面的），應該算是符合用勞力換取金錢吧，雖然之前被騙了50塊有點不爽就是了。
